### PR TITLE
Tell jokes when argument is passed

### PR DESCRIPTION
**Pull request by gokulanv**
#2 Hi @sajoy ,

This is done. Sent you a PR.

### CODE REVIEW

File: console.js [hunk 1]
```diff
@@ -6,7 +6,107 @@
...
+            "Git knows what you did last summer!",
+            "A commit a day keeps the mobs away",
+            "git-outhouse: when even plumbing commands are too nice",
+            "May the --force be with you",
+            "War is peace, freedom is slavery, ignorance is strength, svn is version control",
+            "Subversion to crimestop thoughtcrime and unperson Emmanuel Gitstein",
+            "The git-git-slide is fun to dance to, until it isn't",
+            "In case of fire: git commit, git push, leave the building",
+            "Don't forgit to bring a towel",
+        ]
```
Comment (sajoy) [inline console.js]: Woah! These are great! I don't understand all of them, but I trust you'll remove any jokes that are in bad taste (for example, that make fun of groups of people or serious situations).  Did you source them from anywhere? If you did, what do you think about providing a link to their source - not in the code, but in the README would be good. If you wrote 'em, feel free to list yourself as the source.

Comment (sajoy) [reply console.js]: Also, add a semi-colon here (line 107). Sorry I don't have a linter config set up yet.

File: console.js [hunk 2]
```diff
@@ -6,7 +6,107 @@
...
+        console.log(`${jokes.random()}`)
```
Comment (sajoy) [inline console.js]: Also add a semi-colon here.

**Review by sajoy — CHANGES_REQUESTED** [review]
Looks good @gokul0. Please add the semi-colons and a source for the jokes in the README, then should be good to merge!

File: console.js (hunk 2 above)
Comment (gokulanv) [reply console.js]: Yes, @sajoy I sourced it from a github repository. I will update in Readme.
I will add the semi-colon too.

I have a doubt, How do you set up linter config for a single Js file? Only if this is converted into a project, we can config lint right?

**Comment by gokulanv** [comment]
Hi @sajoy ,

Updated the changes.

**Comment by sajoy** [comment]
Merged. Thanks @gokul0. Also, you're the first contributor! So thanks for that, as well 🎉

**Pull request merged**